packages/api/src/window.ts

```diff
@@ -2271,11 +2344,11 @@ function mapMonitor(m: Monitor | null): Monitor | null {
   return m === null
     ? null
     : {
-        name: m.name,
-        scaleFactor: m.scaleFactor,
-        position: new PhysicalPosition(m.position),
-        size: new PhysicalSize(m.size)
-      }
+      name: m.name,
+      scaleFactor: m.scaleFactor,
+      position: new PhysicalPosition(m.position),
+      size: new PhysicalSize(m.size)
+    }
```

Comment:
let's revert this formatting

Reply:
It was actually done by prettier

Reply:
looks like something is wrong on your setup, as CI was failing because of this